docs/developing.rst

```diff
@@ -0,0 +1,58 @@
+Developing
+==========
+
+Install the requirements
+
+::
+
+    $ pip install -r requirements.txt
+
+(note that lxml, used in the tests, requires that libxslt and libxml2 be
```

Comment:
Ah, right, you keep mentioning that requirement and I keep forgetting. That will get fixed this time! And yeah I'll put that comment in a note box